site/docs/v1/tech/guides/webauthn.adoc

```diff
@@ -0,0 +1,504 @@
+
+image::guides/webauthn/challenge-settings.png[The tenant settings to customize challenge lifetime.,width=1200,role=top-cropped,bottom-cropped]
+
+You may want to adjust the lifetime of these challenges to change how long a user has to complete a ceremony once it has been started. If an attempt is made to complete the ceremony after the challenge expires, it will fail. These values are also used as the timeout for the corresponding https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API[WebAuthn JavaScript API] calls by returning the value in milliseconds as part of the link:/docs/v1/tech/apis/webauthn#start-a-webauthn-passkey-registration[`/api/webauthn/register/start`] and link:/docs/v1/tech/apis/webauthn#start-a-webauthn-passkey-assertion-or-authentication[`/api/webauthn/start`] API responses. The WebAuthn JavaScript APIs treat the timeout value as a hint. The browser may allow the user more time to complete the ceremony (e.g. if the timeout is deemed too short), but FusionAuth will still expire the challenge, preventing the ceremony from being completed. The [field]#WebAuthn authentication# value is used for both authentication and assertion.
+
+== Troubleshooting
+
+Due to the server- and client-side components of each of the WebAuthn ceremonies and compatibility depending on both the browser and operating system, troubleshooting issues can be difficult. These tips may help you when issues come up.
+
+=== Browser Compatibility
+
+WebAuthn is now supported across a variety of web browsers, but if you experience issues, it's worth checking whether your browser is https://caniuse.com/webauthn[compatible]. Be sure to check the notes section for limitations based on specific authenticators or operating systems and evaluate whether you may be experiencing one of those limitations.
+
+=== Secure Context
+
+WebAuthn can only be used in https://w3c.github.io/webappsec-secure-contexts/#secure-contexts[secure contexts]. Briefly, this means that the browser must have a valid TLS connection for the current page. If WebAuthn is called from an `iframe`, then both the `iframe` and the embedding page must be secured.
+
+=== Relying Party Id
+
+A WebAuthn passkey is scoped to a specific relying party Id at the time it is created. Passkeys cannot be used with a different relying party Id than they were registered, and there are restrictions on allowed relying party Ids during a WebAuthn ceremony based on the browser request origin's effective domain.
+
+If the relying party Id for a tenant is changed because of a configuration change or a domain change, passkeys registered prior to the change will no longer work. See link:/docs/v1/tech/admin-guide/webauthn#relying-party-id[Relying Party Id] in the WebAuthn Admin Guide for more detail on constraints and configuration options.
```

Comment:
```suggestion
If the relying party Id for a tenant is changed because of a configuration change or a domain change, passkeys registered prior to the change will no longer work. See the link:/docs/v1/tech/admin-guide/webauthn#relying-party-id[Relying Party Id] section in the WebAuthn Admin Guide for more detail on constraints and configuration options.
```